test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/RazorPagesWebSite.SimpleForms.html

```diff
@@ -0,0 +1,5 @@
+<form><input name="__RequestVerificationToken" type="hidden" value="{0}" /></form>
+<form method="get"></form>
+<form method="post"><input name="__RequestVerificationToken" type="hidden" value="{0}" /></form>
+<form action="/Foo/Bar/Baz.html" method="get"></form>
+<form action="/Foo/Bar/Baz.html" method="post"></form>
```

Comment:
Talked with Damian about this. Since the user specified `action`, they're opting out of our TagHelper ways. Therefore, don't generate a token.